orangecontrib/prototypes/widgets/owsplit.py

```diff
@@ -0,0 +1,135 @@
+import Orange.data
+import numpy as np
+from Orange.data import Table
+from Orange.data.util import SharedComputeValue
+from Orange.widgets import gui
+from Orange.widgets.settings import ContextSetting, DomainContextHandler
+from Orange.widgets.widget import OWWidget, Msg, Output, Input
+from AnyQt.QtCore import Qt
+
+from Orange.data import DiscreteVariable, StringVariable
+from Orange.widgets.utils.itemmodels import DomainModel
+from Orange.widgets.utils.widgetpreview import WidgetPreview
+
+foo = ["foo,bar", "foo,baz", "foo", "bar,baz", "baz,foo", "baz", "bar"]
+
+
+def values_from_string(var_list, splitter):
```

Comment:
If there's no reason to have this as a separate function, I'd make it a `@staticmethod` within the class.